### DISCUSSION TITLE
docker-attestation/

### DISCUSSION THREAD

**Discussion by giscus** [n0vad3v/blog-comments#17] [Ideas]
# docker-attestation/

你有没有遇到过这种情况，对于一个镜像来说，它可以正常 pull 下来：
pad ~ #  docker pull knatnetwork/github-runner-amd64:focal-2.301.1
focal-2.301.1: Pulling from knatnetwork/github-runner-amd64
846c0b181fff: Pull complete 
588b3eef3b63: Pull complete 
189ea0ac146f: Pull complete 
4f4fb700ef54: Pull complete 
546945707c6e: Pull complete 
71464c2d54c9: Pull complete 
1c4efc443e6a: Pull complete 
21bbc223ea9a: Pull complete 
Digest: sha256:6b5b4aa94f8c1e781785e831d18d7ccc1a0de7d70d63b1afd4df3cce27ddd53f
Status: Downloaded newer image for knatnetwork/github-runner-amd64:focal-2.301.1
docker.io/knatnetwork/github-runner-amd64:focal-2.301.1
但是如果你想 inspect 它的 manifest 会发现 no such manifest。

https://nova.moe/docker-attestation/

**Comment by fengzhao** [comment]
你好，仔细阅读了你的排查过程，我是两份dockerfile（只是RUN wget 二进制包有点区别），buildx 都是带了provenance和sbom参数的，分别都构建build x86和arm的image都没问题了，然后合并时也遇到了类似 `docker.io/knatnetwork/github-runner-amd64:focal-2.301.1 is a manifest list ` 这种问题，但是我分别 docker manifest  inpsect 两个镜像时，均已经不存在 unknown 的情况了，但是合并还是有这种`is a manifest list `的情况。

**Reply by fengzhao** [reply]
忽略，我已解决
